test/helpers/ProposalTest.sol

```diff
@@ -153,4 +159,162 @@ abstract contract ProposalTest is PooltogetherGovernorTest {
+    uint256 _snapshot = governorBravo.proposalSnapshot(_proposalId);
+    vm.roll(_snapshot + 1);
+
+    // Ensure the proposal is now Active
+    IGovernor.ProposalState _state = governorBravo.state(_proposalId);
+    assertEq(_state, IGovernor.ProposalState.Active);
+  }
+
+  function _jumpToVotingComplete(uint256 _proposalId) internal {
+    // Jump one block past the proposal voting deadline
+    uint256 _deadline = governorBravo.proposalDeadline(_proposalId);
+    vm.roll(_deadline + 1);
+  }
+
+  function _jumpPastProposalEta(uint256 _proposalId) internal {
+    uint256 _eta = governorBravo.proposalEta(_proposalId);
+    vm.roll(block.number + 1);
+    vm.warp(_eta + 1);
+  }
+
+  function _delegatesVoteOnBravoGovernor(uint256 _proposalId, uint8 _support) internal {
+    require(_support < 3, "Invalid value for support");
+
+    for (uint256 _index = 0; _index < delegates.length; _index++) {
+      vm.prank(delegates[_index].addr);
+      governorBravo.castVote(_proposalId, _support);
+    }
+  }
+
+  function _buildTokenSendProposal(address _token, uint256 _tokenAmount, address _receiver)
+    internal
+    pure
+    returns (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldata,
+      string memory _description
+    )
+  {
+    // Craft a new proposal to send _token.
+    _targets = new address[](1);
+    _values = new uint256[](1);
+    _calldata = new bytes[](1);
+
+    _targets[0] = _token;
+    _values[0] = 0;
+    _calldata[0] =
+      _buildProposalData("transfer(address,uint256)", abi.encode(_receiver, _tokenAmount));
+    _description = "Transfer some tokens from the new Governor";
+  }
+
+  function _submitTokenSendProposalToGovernorBravo(
+    address _token,
+    uint256 _amount,
+    address _receiver
+  )
+    internal
+    returns (
+      uint256 _newProposalId,
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldata,
+      string memory _description
+    )
+  {
+    (_targets, _values, _calldata, _description) =
+      _buildTokenSendProposal(_token, _amount, _receiver);
+
+    // Submit the new proposal
+    vm.prank(PROPOSER);
+    _newProposalId = governorBravo.propose(_targets, _values, _calldata, _description);
+
+    // Ensure proposal is in the expected state
+    IGovernor.ProposalState _state = governorBravo.state(_newProposalId);
+    assertEq(_state, IGovernor.ProposalState.Pending);
+  }
+
+  // Take a proposal through its full lifecycle, from proposing it, to voting on
+  // it, to queuing it, to executing it (if relevant) via GovernorBravo.
+  function _queueAndVoteAndExecuteProposalWithBravoGovernor(
+    address[] memory _targets,
+    uint256[] memory _values,
+    bytes[] memory _calldatas,
+    string memory _description,
+    uint8 _voteType
+  ) internal {
+    // Submit the new proposal
+    vm.prank(PROPOSER);
+    uint256 _newProposalId = governorBravo.propose(
+      _targets, // Go away formatter!
```

Comment:
?

Reply:
Oh I think that is a relic from Gitcoin

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/13/commits/8959eae7ab041cab6ff2c8a1cbaeb0e9d655ffd9